scripts/index-data.ps1

```diff
@@ -19,4 +19,5 @@ npm ci
 npm run build --workspace=indexer
 
 Write-Host 'Running "index-files" CLI tool'
-npx index-files --wait --indexer-url "$env:INDEXER_API_URI" --index-name "$env:AZURE_SEARCH_INDEX" ./data/*.md
+$files = Get-Item "data/*.pdf"
```

Comment:
Should this be *.md?

Reply:
Seems like my previous reply didn't make it :/

Thanks for the catch, yes it should be *.md and I'll fix it